### ISSUE TITLE
systemd-journal-gatewayd doesn't start if DynamicUser option in its .service file is "Yes"

### ISSUE THREAD

**Issue by q2dg** [systemd/systemd#10593]
**systemd version the issue has been seen with**
> 238 
**Used distribution**
 > Fedora 28

Just executing systemctl start systemd-journal-gatewayd should start the service. It throws an error about user permissions. But if I comment the line DynamicUser=yes which is placed in Fedora's regular /lib/systemd/system/systemd-journal-gatewayd file, it starts normally.

EDIT: I've just seen this...: https://github.com/systemd/systemd/pull/7005 Maybe it's related. I had the systemd-journal-gatewayd user created by hand (useradd --system systemd-journal-gatewayd) because I was trying thins before the commenting of problematic line.

**Comment by yuwata** [comment]
Could you provide relevant debug logs about that?

**Comment by q2dg** [comment]
Well...I've installed a fresh Fedora 29 system (which ships systemd v239-6) and I've installed systemd-journal-remote package from Fedora's main repository. After starting it, now systemd-journal-gatewayd service works flawlessly, even having the DynamicUser directive equal to "Yes" in its service unit (which is, in fact, its default value).

For completeness, I've seen that the "systemd-journal-gateway" user appeared in /etc/passwd file before having installed the systemd-journal-remote package and it still remains there after having uninstalled it, so it seems independent from the systemd-journal-remote package (but I haven't found which packet creates this user: rpm -q --scripts systemd doesn't show anything relevant in this case).

So..well...in summary: now I've seen using systemd 239 system-journal-gatewayd service seems to work, i'll close the issue if you don't mind. 
Thanks a lot
